docs/configuration/git-providers/github-cloud.md

```diff
@@ -1,28 +1,22 @@
 # GitHub Cloud
 
-## `values.yaml` configuration
-
-Set your configuration values for your GitHub instance on the `values.yaml` file:
-
-| Field                          | Value                     |
-| ------------------------------ | ------------------------- |
-| `global.github.enabled`        | `true`                    |
-| `global.github.app.name`       | See below how to generate |
-| `global.github.app.id`         | See below how to generate |
-| `global.github.app.privateKey` | See below how to generate |
-
-**Please note that you must go to `http://codacy.example.com/admin/integrations`, select the desired provider and `Test & Save` your configuration for it to be applied.**
-
 ## Create a GitHub App
 
 To integrate with GitHub we use a GitHub App.
 
 Follow the guide to [create the application on GitHub](create-github-app.md).
 
-## Set credentials
+## `values.yaml` configuration
 
-After the application is created, you should copy both the `Client ID` and the `Client Secret` and paste them in the setup page on your Codacy Self-hosted.
+Set your configuration values for the GitHub App on the `values.yaml` file:
 
-![GitHub Application](./images/github-token-retrieval.png)
+| Field                          | Value                     |
+| ------------------------------ | ------------------------- |
+| `global.github.enabled`        | `true`                    |
+| `global.github.app.name`       | See above how to generate |
+| `global.github.app.id`         | See above how to generate |
+| `global.github.app.privateKey` | See above how to generate |
+| `global.github.clientId`       | See above how to generate |
+| `global.github.clientSecret`   | See above how to generate |
```

Comment:
Since these are not generated, they are obtained from GitHub after creating the app. I think this way is more clear
```suggestion
| `global.github.clientId`       | See above how to obtain |
| `global.github.clientSecret`   | See above how to obtain |
```